main/templates/main/post_edit.html

```diff
@@ -1,9 +1,8 @@
 {% extends 'main/base.html' %}
 
 {% block content %}
-    <h1>New post</h1>
```

Comment:
I made it multipurpose for all classes that need form for their work.
So, I've deleted this string and change text in string a bit lower.
I think it's better to have one multipurpose than some only for one need